src/components/pages/parameter-or-variable.jsx

```diff
@@ -34,7 +38,7 @@ const ParameterOrVariablePage = React.createClass({
     }
     return (
       <div>
-        <Link className="btn btn-default" to="/">Retour à l'accueil</Link>
+        <Link className="btn btn-default" onClick={this.handleBackButtonClick} to="/">Retour</Link>
```

Comment:
Good catch :) But the solution you linked needs to be called from a `onClick` handler, or what they call a "reducer" but that's for projects using "redux" library, which is not the case here.

So I adapted your proposition, keeping the `onClick` handler, but using the `history` of the router. It's totally identical *in fine*, but perhaps more elegant.